### DISCUSSION TITLE
Relocating data to a different drive without replication

### DISCUSSION THREAD

**Discussion by MarcusCemes** [minio/minio#12446] [Q&A] [answered]
Hi,

Thank you for the amazing work you've done so far.

After reading the documentation and searching for other posts, I'm still not sure how it is possible to move data from a drive that is to be unmounted and removed.

Our use case is storing large binary files for a backend. We do not need redundancy and the increased storage costs of replication. Our VPS provider allows on-demand storage as fixed-size persistent disks. If we wanted to move to a larger or smaller disk, **is there a way to relocate data to the new drive while both drives are mounted, before taking the old one offline without data loss?** I'm not very familiar with cloud-native and Kubernetes, and it is probably overkilled for our requirements.

Many thanks

**Comment by klauspost** [comment, marked as answer]
@MarcusCemes If you can deal with the downtime you can try to stop the server, move the data and restart. If you feel like it, you can do an initial copy, shut down the server and sync the final changes.

If you can't, simply unmounting the drive and replacing it with a reformatted drive will start restoring the data.

**Reply by MarcusCemes** [reply]
Thank you for getting back to me so quickly. The downtime is a slight deal-breaker, I forgot to mention in the comment above, but the reason we were experimenting with object storage on such a small scale was to decouple data from the physical drives to allow for drive hot-swapping without down-time, like drive pooling, but at a file-level and not filesystem-level.

If downtime is required, it would be easier to just copy the data across without an additional software layer to worry about. When you mentioned restoring data, I imagine this requires replication with at least four instances?
